### ISSUE TITLE
[SCREENREADER, AXE-CORE Serious]: Links must have their visible text as part of their accessible name

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3632]
## Description
**[SCREENREADER, AXE-CORE]: Links must have their visible text as part of their accessible name**

It is confusing to speech input users when they say a visible text label they see, but the speech command does not work because the component's accessible (programmatic) name does not match the visible label. When a user interface component has a visible text label — whether it be real text or an image of text — that text must also be found in the component's accessible (programmatic) name. When the visible label and accessible (programmatic) name for interactive components are in sync, speech input users can effectively interact with those components.

### Element source
```HTML
<a href="https://www.youtube.com/watch?v=oM7oYzL2DCg" aria-label="view information on how you recieved the rating you did on YouTube" target="_blank" rel="noopener noreferrer" title="view information on how you recieved the rating you did on YouTube">Compensation 101: How did I get this rating (YouTube)</a>
```

## Point of Contact

Jennifer

## Acceptance Criteria

As a speech input user I need to be able to navigate the page by speaking the visible text labels of menus, links, and buttons that appear on the screen so I may navigate the content reliably.

## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

Run axe-coconut scan, as listed in the [accessibility best practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md#automated-accessibility508-testing)

## Possible Fixes (optional)

Fix this issue by ensuring that the accessible name — such as the aria-label — contains the text of the visible label in the same order in which it appears. While it is not necessary that the visible label and accessible name match exactly, it is a best practice to have the accessible name begin with the visible text.

```HTML
<a href="https://www.youtube.com/watch?v=oM7oYzL2DCg" aria-label="Compensation 101: How did I get this rating (YouTube)" target="_blank" rel="noopener noreferrer" title="View information on how you received the rating you did on YouTube">Compensation 101: How did I get this rating (YouTube)</a>
```

## WCAG or Vendor Guidance (optional)

[Label and name from content mismatch](https://dequeuniversity.com/rules/axe/3.4/label-content-name-mismatch?application=coconut)

**Comment by jason-gcio** [comment]
Addressed in this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/4013
